{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/urls.py

```diff
@@ -2,6 +2,7 @@
 
 from . import views
 
+app_name = 'users'
```

Comment:
Any comments about `app_name` on `urls` module level? Why is it necessary?. Maybe I missed some django changes, but I really do not remember that.

Reply:
It's optional in Django 2.0, because `include` no longer supports `app_name`:

https://docs.djangoproject.com/en/2.0/topics/http/urls/#url-namespaces-and-included-urlconfs
https://docs.djangoproject.com/en/2.0/releases/2.0/#features-removed-in-2-0

Reply:
@zoidbergwill is right, it's a combination of several removed features:

> - Support for setting a URL instance namespace without an application namespace is removed.

This URL pattern is included with namespace, so now we must provide an `app_name`. At the same time:

> - The app_name argument to include() is removed.